internal/MozziGuts.hpp

```diff
@@ -126,15 +131,15 @@ __attribute__((noinline)) void adcStartReadCycle() {
 	}
 }
 
-int mozziAnalogRead(uint8_t pin) {
+int16_t mozziAnalogRead(uint8_t pin) {
 	pin = adcPinToChannelNum(pin); // allow for channel or pin numbers; on most platforms other than AVR this has no effect. See note on pins/channels
 	adc_channels_to_read.push(pin);
 	return analog_readings[channelNumToIndex(pin)];
 }
 
 #if !MOZZI_IS(MOZZI_AUDIO_INPUT, MOZZI_AUDIO_INPUT_NONE)
 static AudioOutputStorage_t audio_input; // holds the latest audio from input_buffer
-AudioOutputStorage_t getAudioInput() { return audio_input; }
+int16_t getAudioInput() { return audio_input; }
```

Comment:
Idem `uint16_t`?

Reply:
Done.

Reply:
I think l. 134 and l.142 should also be unsigned?

Reply:
Yes, indeed. And of course line 141. I hope we've covered everything, now...